packages/appstore/src/components/main-title-bar/index.tsx

```diff
@@ -29,6 +30,11 @@ const MainTitleBar = () => {
         setActiveIndex(selected_region === 'Non-EU' ? 0 : 1);
     }, [selected_region]);
 
+    const { is_wallet_enabled } = useFeatureFlags();
+    const { has_wallet } = useWalletsList();
+
+    const should_show_banner = is_wallet_enabled && !has_wallet;
```

Comment:
As discussed with @mahdiyeh-deriv and @thisyahlen-deriv, We need to also add the `status` of wallet migration to this check.

Reply:
Yes @farzin-deriv, we should do this before we merge to master @heorhi-deriv 🙏

Reply:
ok guys, will do

Reply:
hey @farzin-deriv @thisyahlen-deriv @mahdiyeh-deriv not fully understand which status you want to add guys.

here we are checking that FF is enabled and if there is no wallet we show the banner (inside the banner we have logic that show us proper ui based on status). so why we need to add status check in this condition:
` const should_show_banner = is_wallet_enabled && !has_wallet` ?

Reply:
do you want to add smth like:
```
const is_ineligible = useWalletMigration()
const should_show_banner = is_wallet_enabled && !has_wallet && !is_ineligible
```
?

Reply:
@heorhi-deriv, lets do this instead
`    const { is_eligible } = useWalletMigration();`
` {is_eligible && <WalletsBanner />}`
This doesnt show the banner even if the status is not there or other than `is_eligible`
OR
We can check if `data` exists from `useWalletMigration` hook, then show the banner